app/src/main/java/ar/com/wolox/android/cookbook/tests/TestLoginRecipeService.kt

```diff
@@ -0,0 +1,17 @@
+package ar.com.wolox.android.cookbook.tests
+
+import ar.com.wolox.wolmo.core.di.scopes.ApplicationScope
+import javax.inject.Inject
+
+@ApplicationScope
+open class TestLoginRecipeService @Inject constructor() {
```

Comment:
This should perform an async operation to better emulate the behavior of an API, for example.